terraform/050-pw-manager/README.md

```diff
@@ -1,13 +1,13 @@
 # 050-pw-manager - ECS service for password manager API and S3 config for UI
-This module is used to create an ECS service running the password manager API and static site hosting for the UI.
+This module is used to create an ECS service running the password manager API.
+
+The password manager UI can be deployed using the [silinternatonal/pages/cloudflare](https://registry.terraform.io/modules/silinternational/pages/cloudflare/latest) module.
 
 ## What this does
 
  - Create ALB target group for API with hostname based routing
  - Create task definition and ECS service for password manager API service
- - Create S3 bucket for UI
- - Create CloudFront distribution to provide SSL support for UI
- - Create Cloudflare DNS records for API and UI
+ - Create Cloudflare DNS record
```

Comment:
I know it's somewhat implied, but do you think it would be worth explicitly stating that this is the DNS record for the API?